Tools/Carbonix_scripts/process_sitl_defaults.py

```diff
@@ -39,11 +39,23 @@ def process_defaults(file, depth=0):
             continue
 
         if line.startswith("@delete"):
-            pattern = line.split(maxsplit=1)[1]
-            for i in range(len(param_list)):
-                param_name = re.split(r"[\s,]+", param_list[i])[0]
+            # Strip trailing comments
+            line = re.sub(r"\s*#.*$", "", line)
+            line.strip()
```

Comment:
shouldn't it be line = line.strip()

Reply:
Yes, good catch

Reply:
This line was actually unnecessary, which is why I didn't catch that it wasn't doing what it was supposed to. The `split` already harmlessly removes trailing split characters.